package.json

```diff
@@ -26,6 +26,14 @@
     ".": {
       "require": "./dist/cjs/index.js",
       "import": "./dist/esm/index.js"
+    },
+    "./*": {
+      "require": "./dist/cjs/locales/*/index.js",
+      "import": "./dist/esm/locales/*/index.js"
+    },
+    "./*/*": {
```

Comment:
What is this for? I don't think we have index.ts in these sub-directories. 

If it's necessary, lets add that later.

Reply:
I had to add these to be able to import the individual locales without importing the entire library. Each locale has an `index.ts` that builds to `/dist/cjs/<locale>/index.js` and `/dist/esm/<locale>/index.js` respectively:

![image](https://user-images.githubusercontent.com/104778230/234423842-59e84e06-fa5c-4dce-adee-e716942bf193.png)

So by adding these it allows all locales to be imported individually either as CJS or ESM: 
```jsonc
// import all of chrono, like normal 
".": {
  // const chrono = require('chrono-node')
  "require": "./dist/cjs/index.js",
  // import chrono from 'chrono-node'
  "import": "./dist/esm/index.js"
},
// import only one locale (`en`)
"./*": {
  // const chrono = require('chrono-node/en')
  "require": "./dist/cjs/locales/*/index.js",
  // import chrono from 'chrono-node/en'
  "import": "./dist/esm/locales/*/index.js"
},
// import only one locale, with subtype (`zh/hant`)
"./*/*": {
  // const chrono = require('chrono-node/zh/hant')
  "require": "./dist/cjs/locales/*/*/index.js",
  // import chrono from 'chrono-node/zh/hant'
  "import": "./dist/esm/locales/*/*/index.js"
}
```

Reply:
I see. I forgot about `/zh/hant`.